nvflare/fuel/utils/component_classes.json

```diff
@@ -1 +1,572 @@
+        "nvflare.app_common.workflows.broadcast_and_process"
+    ],
+    "BroadcastAndWait": [
+        "nvflare.app_common.workflows.broadcast_operator"
+    ],
+    "CrossSiteEval": [
+        "nvflare.app_common.workflows.cross_site_eval"
+    ],
+    "CrossSiteModelEval": [
+        "nvflare.app_common.workflows.cross_site_model_eval"
+    ],
+    "Cyclic": [
+        "nvflare.app_common.workflows.cyclic"
+    ],
+    "CyclicController": [
+        "nvflare.app_common.workflows.cyclic_ctl"
+    ],
+    "ErrorHandlingController": [
+        "nvflare.app_common.workflows.error_handling_controller"
+    ],
+    "FedAvg": [
+        "nvflare.app_common.workflows.fedavg"
+    ],
+    "GlobalModelEval": [
+        "nvflare.app_common.workflows.global_model_eval"
+    ],
+    "HierarchicalStatisticsController": [
+        "nvflare.app_common.workflows.hierarchical_statistics_controller"
+    ],
+    "InitializeGlobalWeights": [
+        "nvflare.app_common.workflows.initialize_global_weights"
+    ],
+    "ModelController": [
+        "nvflare.app_common.workflows.model_controller"
+    ],
+    "Scaffold": [
+        "nvflare.app_common.workflows.scaffold"
+    ],
+    "ScatterAndGather": [
+        "nvflare.app_common.workflows.scatter_and_gather"
+    ],
+    "ScatterAndGatherScaffold": [
+        "nvflare.app_common.workflows.scatter_and_gather_scaffold"
+    ],
+    "SplitNNController": [
+        "nvflare.app_common.workflows.splitnn_workflow"
+    ],
+    "StatisticsController": [
+        "nvflare.app_common.workflows.statistics_controller"
+    ],
+    "CCManager": [
+        "nvflare.app_opt.confidential_computing.cc_manager"
+    ],
+    "FlowerClientApplet": [
+        "nvflare.app_opt.flower.applet"
+    ],
+    "FlowerServerApplet": [
+        "nvflare.app_opt.flower.applet"
+    ],
+    "FlowerClientConnector": [
+        "nvflare.app_opt.flower.connectors.flower_connector"
+    ],
+    "FlowerServerConnector": [
+        "nvflare.app_opt.flower.connectors.flower_connector"
+    ],
+    "HECrossSiteModelEval": [
+        "nvflare.app_opt.he.cross_site_model_eval"
+    ],
+    "HEInTimeAccumulateWeightedAggregator": [
+        "nvflare.app_opt.he.intime_accumulate_model_aggregator"
+    ],
+    "HEModelDecryptor": [
+        "nvflare.app_opt.he.model_decryptor"
+    ],
+    "HEModelEncryptor": [
+        "nvflare.app_opt.he.model_encryptor"
+    ],
+    "HEModelSerializeFilter": [
+        "nvflare.app_opt.he.model_serialize_filter"
+    ],
+    "HEModelShareableGenerator": [
+        "nvflare.app_opt.he.model_shareable_generator"
+    ],
+    "PTClientAPILauncherExecutor": [
+        "nvflare.app_opt.pt.client_api_launcher_executor"
+    ],
+    "PTFedAvgEarlyStopping": [
+        "nvflare.app_opt.pt.fedavg_early_stopping"
+    ],
+    "PTFedOptModelShareableGenerator": [
+        "nvflare.app_opt.pt.fedopt"
+    ],
+    "FedOpt": [
+        "nvflare.app_opt.pt.fedopt_ctl",
+        "nvflare.app_opt.tf.fedopt_ctl"
+    ],
```

Comment:
How do users determine between PT vs TF?

Reply:
for the same class name, users will get an error message when using "name". They have to change to use "path".